packages/@ourworldindata/grapher/src/core/Grapher.tsx

```diff
@@ -1292,7 +1307,7 @@ export class Grapher
         let variablesDataMap: MultipleOwidVariableDataDimensionsMap
 
         const startMark = performance.now()
-        if (this.isAdmin) {
+        if (this.isAdminObjectAvailable) {
```

Comment:
Here we care about the `window.admin` object, which we then use in the nested calls rather than about whether we are in the admin, which is currently not the same thing.